nucypher/blockchain/eth/sol/source/contracts/proxy/README.MD

```diff
@@ -30,21 +30,9 @@ New versions of contract can expand values, but must contain all old values (fir
 This contract is like library because it's storage is not used.
 If client send request to the contract without using dispatcher then request could be executed without exception
 but using wrong target address (should be dispatcher address) and wrong storage (should be dispatcher storage).
-* Government - contract that organizes voting for upgrade of contracts and carries out this update.
-
-# Interaction order
-* Admin creates contract_v1 and creates dispatcher using contract_v1 address
-* Client instantiates contract_v1 using dispatcher address and sends requests
-* Admin or any client creates contract_v2 and creates voting in the government contract
-* All clients vote for or against upgrade during the voting time
-* After successful voting admin or any client commits upgrade in the government contract
-* Client uses old instance without changing (if the methods are not changed)
 
 # Development
```

Comment:
Where do we want this documentation to be published?  It seems isolated from the rest of the codebase

Reply:
It's really old docs, maybe move it to wiki or remove it?